core/src/com/unciv/CrashScreen.kt

```diff
@@ -0,0 +1,160 @@
+package com.unciv
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.Actor
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.badlogic.gdx.utils.Align
+import com.unciv.models.ruleset.RulesetCache
+import com.unciv.ui.utils.*
+import java.io.PrintWriter
+import java.io.StringWriter
+
+/** Screen to crash to when an otherwise unhandled exception or error is thrown. */
+class CrashScreen(message: String): BaseScreen() {
```

Comment:
Much better!
Still, this function is about 25% longer than in needs to be (in lines) because of all the unnecessary 'enter's, which makes it harder to parse
So please remove gratuitous line breaks :)

Reply:
Funny. I actually thought those would improve readability by separating different elements… IIRC My first/only other GH contributions were to a project with lots of line breaks— I think I went overboard with putting every first function's argument on its own line.